README.md

```diff
@@ -132,19 +133,27 @@ The API documentation can be found [here](http://docs.openshift.org/latest/rest_
 FAQ
 ---
 
-1. How does OpenShift relate to Kubernetes?
+1. How does Origin relate to Kubernetes?
 
-    OpenShift embeds Kubernetes and adds additional functionality to offer a simple, powerful, and
+    Origin embeds Kubernetes and adds additional functionality to offer a simple, powerful, and
     easy-to-approach developer and operator experience for building applications in containers.
-    Kubernetes today is focused around composing containerized applications - OpenShift adds
+    Kubernetes today is focused around composing containerized applications - Origin adds
     building images, managing them, and integrating them into deployment flows.  Our goal is to do
-    most of that work upstream, with integration and final packaging occurring in OpenShift.  As we
+    most of that work upstream, with integration and final packaging occurring in Origin.  As we
     iterate through the next few months, you'll see this repository focus more on integration and
     plugins, with more and more features becoming part of Kubernetes.
 
-2. What can I run on OpenShift?
+2. How does Atomic Enterprise relate to Origin and OpenShift?
```

Comment:
This paragraph works, but the previous one is weak.  It highlights how this naming setup is confused (origin as a technology) and distinct from product.  I would make the previous paragraph about technology (enterprise distribution of Kube), and this is just a paragraph about product - describe AEP and OpenShift as two products that exist on top of the technology.  Don't include source to image here.